scripts/InstallHootDevelopmentEnvironment.sh

```diff
@@ -43,7 +43,7 @@ fi
 # use tomcat 7 here instead of tomcat 6 (what's tested with hoot...I don't really like this...but), since eclipse 
 # installed from repos (installed in this script later) needs tomcat 7
```

Comment:
I agree....I hadn't done anything with vagrant before this task so was reluctant to try, but think I could do it now.   I'm the only one who's ever this dev env script...just added it in case it was useful to anyone else.  But as the script is now, it requires extra updating whenever the vagrant provision script is updated, which is no good.